WRN-12918 : Fixed "debug spotlight" option to work in samplers (#2989)
enactjs/enact: packages/spotlight/CHANGELOG.md

The following is a curated list of changes in the Enact spotlight module, newest changes on the top.

## [unreleased] 

### Fixed

> - `spotlight/styles/debug.less` to show the debug box when each theme sampler's `debug spotlight`option is set to `true`

Reviewer: ```suggestion
- Styles for `debug spotlight` option in samplers to work properly
```
